localtests/latin1text-to-utf8mb4-insert/create.sql

```diff
@@ -0,0 +1,9 @@
+drop table if exists gh_ost_test;
+create table gh_ost_test (
+  id int auto_increment,
+  t text charset latin1 collate latin1_swedish_ci,
+  primary key(id)
+) auto_increment=1 charset latin1 collate latin1_swedish_ci;
+
+insert into gh_ost_test values (null, char(189));
```

Comment:
189 and 190 chosen specifically because they can be replaced with a utf8 character equivalent (prefixed with `\xC2`). Thus the checksum at the end of the test will pass.